src/sandbox/sandbox_linux.go

```diff
@@ -68,7 +68,10 @@ func Sandbox(args []string) error {
 		}
 	}
 	err = syscall.Exec(cmd, args, env)
-	return fmt.Errorf("Failed to exec %s: %s", cmd, err)
+	if err != nil {
+		return fmt.Errorf("Failed to exec %s: %s", cmd, err)
+	}
+	return nil
```

Comment:
not sure how this got in here :disappointed:

Reply:
Guessing there's a newer linter now that didn't used to check this.

I think it's wrong in this case though, `Exec` only ever returns on failure so the code before was legit.

Reply:
I guess the linter doesn't know that. This shuts it up so I think it's fine.

Reply:
Yeah IIRC there are ways of marking this kind of thing in C compilers but it's not something Go knows about. I guess it's not a big deal, although maybe that linter could special-case one or two functions like this.